internal/dialect/dialectquery/ydb.go

```diff
@@ -0,0 +1,62 @@
+package dialectquery
+
+import "fmt"
+
+type Ydb struct{}
+
+var _ Querier = (*Ydb)(nil)
+
+func (c *Ydb) CreateTable(tableName string) string {
+	return fmt.Sprintf(`
+		CREATE TABLE %s (
```

Comment:
Where possible, we try to add NOT NULL and a default. Would something like this make sense for ydb?

```sql
CREATE TABLE goose_db_version (
    version_id Uint64 NOT NULL,
    is_applied Bool NOT NULL,
    tstamp Timestamp DEFAULT CURRENT_TIMESTAMP, 
    PRIMARY KEY (version_id)
);
```

Nits,

1. Remove the trailing semicolons from all queries
2. Use the q = `<query>` and `return fmt.Sprintf(q, tableName)` pattern like all the other dialect queries (this makes it easier to do maintain, do large search/replace, etc.)

Reply:
YDB not supports `DEFAULT` and `NOT NULL` constraints for non-pk columns in latest release (23.3). We will do that in one of the next release. With back-compatible of course. But in current release we can do autocomplete tstamp field explicitly on insert stage with built-in function `CurrentUtcTimestamp()`